build-scripts/ios/gen_build_config.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+const config = require('../../config.json');
```

Comment:
Environment variable seems like a decent solution, we can use travis secrets then.